src/Cache/FileCacheAdapter.php

```diff
@@ -32,6 +30,9 @@
      * @var string control characters for keys, reserved by PSR-16
      */
     const PSR16_RESERVED = '/\{|\}|\(|\)|\/|\\\\|\@|\:/u';
+    const FILE_PREFIX = 'pdp::';
+    const FILE_EXTENSION = '.cache';
+    const CACHE_TTL = 86400 * 365 * 5;
```

Comment:
@jeremykendall this is the only parameter that needs consideration IMHO put whatever value you want I'm not worry about this could be 1 WEEK or 1 DAY... I've put 5 years just that you could change the value to one witch is more appropriate